src/LaraFormsBuilderServiceProvider.php

```diff
@@ -8,6 +8,18 @@
 
 class LaraFormsBuilderServiceProvider extends PackageServiceProvider
 {
+    public function boot(): void
```

Comment:
We shouldn't override the boot function, the only thing we should do is add `->hasAssets()` in the `configurePackage` function and move the `css` folder to `dist`